packages/bitcore-wallet-service/src/lib/chain/eth/index.ts

```diff
@@ -93,10 +93,27 @@ export class EthChain implements IChain {
       if (err) {
         return cb(err);
       }
+      // getPendingTxs returns all txps when given a native currency
       server.getPendingTxs(opts, (err, txps) => {
         if (err) return cb(err);
-        // Do not lock eth multisig amount
-        const lockedSum = opts.multisigContractAddress ? 0 : _.sumBy(txps, 'amount') || 0;
+        let fees = 0;
+        let amounts = 0;
+
+        txps = txps.filter(txp => {
+          // Add gas used for tokens when getting native balance
+          if (!opts.tokenAddress) {
+            fees += txp.fee;
```

Comment:
Might need to `|| 0` here. Or are we 100% certain that all txps are going to have fee populated?